README.md

```diff
@@ -40,7 +40,9 @@ gcloud container --project {your-project-name} clusters create "spring-test" --z
 gcloud config set container/cluster spring-test
 gcloud config set compute/zone us-central1-b
 gcloud container clusters get-credentials spring-test
+kubectl version
 ```
+> :information_source: the last command causes the access token to be generated and saved to the kubeconfig file - it can be any valid kubectl command
```

Comment:
So this was a "fun" one... I was banging my head wondering why the auth would work sometime and not others. We have to actually run `kubectl` with a command that requires auth in order for the auth token to be written to the kubeconfig file. To see the proof, do the following:

```java
gcloud container clusters get-credentials spring-test
kubectl config view
```

> ℹ️  note that `kubectl config view` does not suffice to get the token generated but `kubectl version` does.

You will notice there is no "access-token" entry in the "users" section for the "spring-test" cluster - the end of the kubeconfig file will look similar to:
```yaml
- name: gke_bono007-sandbox_us-central1-b_spring-test
  user:
    auth-provider:
      config:
        cmd-args: config config-helper --format=json
        cmd-path: /Users/cbono/indeed/spring.io/scdf/google-cloud-sdk/bin/gcloud
        expiry-key: '{.credential.token_expiry}'
        token-key: '{.credential.access_token}'
      name: gcp
```
and your tests will fail auth.

Now execute the following commands:
```java
kubectl version
kubectl config view
```
You will notice there is now an "access-token" entry in the "users" section for the "spring-test" cluster - the end of the kubeconfig file will now look similar to:
```yaml
- name: gke_bono007-sandbox_us-central1-b_spring-test
  user:
    auth-provider:
      config:
        access-token: <redacted>
        cmd-args: config config-helper --format=json
        cmd-path: /Users/cbono/indeed/spring.io/scdf/google-cloud-sdk/bin/gcloud
        expiry: "2021-12-26T04:55:03Z"
        expiry-key: '{.credential.token_expiry}'
        token-key: '{.credential.access_token}'
      name: gcp
```
and your tests will now pass.